utils/tests/utils.go

```diff
@@ -134,6 +134,7 @@ func ValidateListsIdentical(expected, actual []string) error {
 		return fmt.Errorf("unexpected behavior, \nexpected: [%s], \nfound:    [%s]", strings.Join(expected, ", "), strings.Join(actual, ", "))
 	}
 	err := compare(expected, actual)
+	log.Info(fmt.Errorf("unexpected behavior, \nexpected: [%s], \nfound:    [%s]", strings.Join(expected, ", "), strings.Join(actual, ", ")))
```

Comment:
If we'd like to log this information, we should add to the error returned (concatenating this message to the original error), but not log it as an info log message.